android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -476,145 +476,176 @@ public void run() {
 		}
 	}
 
-	protected MotionEvent convertTouchEvent(OnTouchEvent touchEvent){
-		List<TouchEvent> eventList = touchEvent.getEvent();
-		if (eventList == null || eventList.size() == 0) return null;
+	List<MotionEvent> convertTouchEvent(OnTouchEvent onTouchEvent){
+		List<MotionEvent> motionEventList = new ArrayList<MotionEvent>();
 
-		TouchType touchType = touchEvent.getType();
-		if (touchType == null){ return null;}
+		List<TouchEvent> touchEventList = onTouchEvent.getEvent();
+		if (touchEventList == null || touchEventList.size() == 0) return null;
 
-		int eventListSize = eventList.size();
+		TouchType touchType = onTouchEvent.getType();
+		if (touchType == null) { return null; }
 
-		MotionEvent.PointerProperties[] pointerProperties = new MotionEvent.PointerProperties[eventListSize];
-		MotionEvent.PointerCoords[] pointerCoords = new MotionEvent.PointerCoords[eventListSize];
+		if(sdlMotionEvent == null) {
+			if (touchType == TouchType.BEGIN) {
+				sdlMotionEvent = new SdlMotionEvent();
+			} else{
+				return motionEventList;
```

Comment:
Why is an empty array passed here instead of null?

Reply:
Theoretically, the else shouldn't be hit. It makes more sense to return null though. I will update that.